src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -382,6 +404,9 @@ public boolean isPaired() {
 
     @Override
     public void setIsPaired( final boolean isPaired ) {
+        // Setting isPaired==false changes numberOfReads to 1, but the GenomicsConverter will
```

Comment:
I opened a HB issue for this a while back at David's request so we can review to determine how to proceed. #891